instrumentation/trilogy/lib/opentelemetry/instrumentation/trilogy/patches/client.rb

```diff
@@ -131,6 +132,10 @@ def database_name
             connection_options[:database]
           end
 
+          def database_user
```

Comment:
Is it reasonable to be concerned with adding very generic helper method names to our instrumentation, do we worry about overriding an existing or potentially to be added method in the future?

Reply:
It is reasonable to be concerned. I worry about all the helper methods here, but I convinced myself I'm not making the situation worse. I would like to find a cleaner way to implement these helpers in future.

Reply:
I think this is reasonable and potentially solvable with `def _otel_database_user` or some such thing.

Reply:
I plan to tackle this in a follow-on PR. I opened #563 to track it.